<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="Scroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>540</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>540</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background-image: url(resources/1.11.png);
}</string>
     </property>
     <widget class="QLabel" name="Year">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>376</y>
        <width>300</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Song">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>326</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Album">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>401</y>
        <width>300</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Genre">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>426</y>
        <width>300</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Path">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>476</y>
        <width>300</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Artist">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>351</y>
        <width>300</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="Photo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
 border: none;
 border-radius: 20px;
 color: white;
 background: none;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Left">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>270</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 16px;
 color: white;
 background: none;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/Left.png</normaloff>resources/Left.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Right">
      <property name="geometry">
       <rect>
        <x>575</x>
        <y>270</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 16px;
 color: white;
 background: none;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/Right.png</normaloff>resources/Right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Change">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 15px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Dir">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 15px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>New song</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>resources/Rectangle 69.png</pixmap>
      </property>
     </widget>
     <widget class="QCheckBox" name="Color_set">
      <property name="geometry">
       <rect>
        <x>535</x>
        <y>22</y>
        <width>55</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator:checked{
	background-color: #787878;
    image: url(resources/cat.png);
}

QCheckBox::indicator:unchecked{
	background-color: #787878;
    image: url(resources/purple.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Number">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>451</y>
        <width>300</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>Year</zorder>
     <zorder>Song</zorder>
     <zorder>Album</zorder>
     <zorder>Genre</zorder>
     <zorder>Path</zorder>
     <zorder>Artist</zorder>
     <zorder>Photo</zorder>
     <zorder>Left</zorder>
     <zorder>Right</zorder>
     <zorder>Change</zorder>
     <zorder>Dir</zorder>
     <zorder>Color_set</zorder>
     <zorder>Number</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background-image: url(resources/2.2.png);
}</string>
     </property>
     <widget class="QPushButton" name="PhotoEd">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>161</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 20px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>New image</string>
      </property>
     </widget>
     <widget class="QLabel" name="PhotoEdRes">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>380</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="PathEd">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>161</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 20px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>Move to ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ArtistEd">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>165</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
 border:  1px solid white;
 border-radius: 15px;
 color: white;
 background: none;
padding:0px 10px 0px 10px;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="placeholderText">
       <string>artist ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="AlbumEd">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>215</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
 border:  1px solid white;
 border-radius: 15px;
 color: white;
 background: none;
padding:0px 10px 0px 10px;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="placeholderText">
       <string>album ...</string>
      </property>
     </widget>
     <widget class="QLabel" name="PathEdRes">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Accept">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 20px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SongEd">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>165</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
 border:  1px solid white;
 border-radius: 15px;
 color: white;
 background: none;
padding:0px 10px 0px 10px;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="placeholderText">
       <string>song ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GenreEd">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>215</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
 border:  1px solid white;
 border-radius: 15px;
 color: white;
 background: none;
padding:0px 10px 0px 10px;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="placeholderText">
       <string>genre ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="YearEd">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>265</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
 border:  1px solid white;
 border-radius: 15px;
 color: white;
 background: none;
padding:0px 10px 0px 10px;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="placeholderText">
       <string>year ...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Back_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 15px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>resources/Rectangle 69.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="NumberEd">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>265</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
 border:  1px solid white;
 border-radius: 15px;
 color: white;
 background: none;
padding:0px 10px 0px 10px;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="placeholderText">
       <string>track number ...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>88</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Change meta</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>PhotoEd</zorder>
     <zorder>PhotoEdRes</zorder>
     <zorder>PathEd</zorder>
     <zorder>ArtistEd</zorder>
     <zorder>AlbumEd</zorder>
     <zorder>PathEdRes</zorder>
     <zorder>Accept</zorder>
     <zorder>SongEd</zorder>
     <zorder>GenreEd</zorder>
     <zorder>YearEd</zorder>
     <zorder>Back_2</zorder>
     <zorder>NumberEd</zorder>
     <zorder>label_4</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">QWidget
{
background-image: url(resources/2.3.png);
}</string>
     </property>
     <widget class="QTableWidget" name="Songtab">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>150</y>
        <width>510</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget::item:selected { 
color: #787878; 
background-color: white;
}

QTableWidget
{
 border: none;
 border-radius: 16px;
 color: white;
 background: none;
 border-bottom: none;
}
</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>505</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="ChDir">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>470</y>
        <width>45</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 16px;
 color: white;
 background: none;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/folder.png</normaloff>resources/folder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="SortType">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox
{
 	border: 1px solid white;
 	border-radius: 13px;
 	color: white;
 	background: #787878;
      padding:0px 10px 0px 10px;
}

QComboBox::drop-down 
{
    border: 0px; /* This seems to replace the whole arrow of the combo box */
}

/* Define a new custom arrow icon for the combo box */
QComboBox::down-arrow {
    image: url(./resources/arrow.png);
    width: 17px;
    height: 17px;
    margin-right: 15px;
}</string>
      </property>
      <item>
       <property name="text">
        <string>Ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Descending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Song</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Artist</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Album</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Genre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Year</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Number</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 15px;
 color: #787878;
 background: white;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SortBut">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 16px;
 color: white;
 background: none;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/sort.png</normaloff>resources/sort.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>29</width>
        <height>29</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="Search">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QLineEdit
{
 border: 1px solid white;
 border-radius: 13px;
 color: white;
 background: #787878;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchBut">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 border: none;
 border-radius: 16px;
 color: white;
 background: none;
 border-bottom: none;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/search.png</normaloff>resources/search.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>29</width>
        <height>29</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>resources/Rectangle 69.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>86</y>
        <width>191</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
color: white;
background-color: rgba(0, 0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Choose a song</string>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>Songtab</zorder>
     <zorder>ChDir</zorder>
     <zorder>SortType</zorder>
     <zorder>Back</zorder>
     <zorder>SortBut</zorder>
     <zorder>Search</zorder>
     <zorder>SearchBut</zorder>
     <zorder>label_5</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
